src/test/java/NumbersProcessor/logic/NumbersProcessorTest.java

```diff
@@ -45,18 +44,18 @@
         return resultList;
     }
 
-    @org.junit.Test
-    public void filterNumberStringsTest() {
+    @Test
```

Comment:
@test and other annotated methods should be at top of class method structure.
Helper methods like readFileToCreateListForTest after them.